host-interaction/registry/query/query-registry-key.yml

```diff
@@ -3,13 +3,25 @@ rule:
     name: query registry key
     namespace: host-interaction/registry/query
     author: michael.hunhoff@fireeye.com
-    scope: function
+    scope: basic block
     att&ck:
       - Discovery::Query Registry [T1012]
     mbc:
       - Operating System::Registry::Query Registry Key [C0036.005]
     examples:
       - 493167E85E45363D09495D0841C30648:0x404930
+      - B5F85C26D7AA5A1FB4AF5821B6B5AB9B:0x402608
   features:
     - or:
+      - api: advapi32.RegEnumKey
```

Comment:
I think enum and query provide different functionality. Is it worth splitting these up?

Reply:
I see both enum and query as providing information about a given key, in the case of enum the name of one subkey each time it is called, so IMO it's not worth creating a separate rule just to split these up. But if you think they should still be split I'd be happy to update the PR.

Reply:
A few descriptions from MSDN. Maybe we change to `query or enumerate registry key`? 😄 

- Enumerates the subkeys of the specified open registry key. The function retrieves the name of one subkey each time it is called.
- The ZwQueryKey routine provides information about the class of a registry key, and the number and sizes of its subkeys.
- Retrieves information about the specified registry key.

Reply:
renamed to `query or enumerate registry key`.